Src/Orbiter/Vessel.h

```diff
@@ -546,8 +546,29 @@ class Vessel: public VesselBase {
 	}
 	// return the average thrust level for a thruster group
 
+	inline double GetThrusterGroupLevel_User() const
+	{
+		double level = 0.0;
+		int    number = 0;;
+		ThrustGroupSpec* tgs;
+
+		for (DWORD i = 0; i < nthruster_grp_user; i++)
+		{
+			tgs = thruster_grp_user[i];
+			for (DWORD ii = 0; ii < tgs->nts; ii++)
```

Comment:
Martin can weigh in further, but I find `ii` easy to mix up with `i` when scanning the code in the inner loop here. What about naming the `ii` variable something like `thrusterIndex` so it's more descriptive?